src/slider/stateful-slider-container.js

```diff
@@ -22,28 +22,23 @@ class StatefulSliderContainer extends React.Component<
   StateT,
 > {
   static defaultProps = {
-    initialState: {value: []},
+    initialState: {values: []},
     stateReducer: defaultStateReducer,
     onChange: () => {},
   };
 
   state = {...this.props.initialState};
 
-  onChange = (config: {event: *, value: Array<number>}) => {
-    const {event, ...params} = config;
-    this.internalSetState(STATE_CHANGE_TYPE.change, event, params);
+  onChange = (params: {values: Array<number>}) => {
+    this.internalSetState(STATE_CHANGE_TYPE.change, params);
     const {onChange} = this.props;
-    return onChange({event, ...params});
+    return onChange({...params});
   };
 
-  internalSetState = (
-    type: ChangeActionT,
-    e: SyntheticInputEvent<HTMLInputElement>,
-    {value}: ParamsT,
-  ) => {
-    const nextState = {value};
+  internalSetState = (type: ChangeActionT, {values}: ParamsT) => {
+    const nextState = {values};
     const {stateReducer} = this.props;
-    const newState = stateReducer(type, nextState, this.state, e);
+    const newState = stateReducer(type, nextState, this.state);
```

Comment:
may want to call out in the migration documentation that we are no longer passing the event